util/proposal.go

```diff
@@ -5,12 +5,18 @@ import (
 	"bytes"
 	"encoding/base64"
 	"regexp"
+	"strconv"
 
 	www "github.com/decred/politeia/politeiawww/api/v1"
 )
 
+const (
+	minNameLength = www.PolicyMinProposalNameLength
+	maxNameLength = www.PolicyMaxProposalNameLength
+)
+
 var (
-	validProposalName = regexp.MustCompile(www.ValidProposalNameRegExp)
+	supportedChars = www.PolicyProposalNameSupportedCharacters
```

Comment:
This variable is unnecessary, just reference `www.PolicyProposalNameSupportedCharacters` directly.